src/tools/gxpkg.ss

```diff
@@ -294,24 +303,31 @@
       (delete-file dest))))
 
 (def (pkg-build pkg (dependents? #t))
-  (if (equal? pkg "all")
+  (cond
+   ((equal? pkg "all")
```

Comment:
I can't name my package "all"? Fascist! Is that documented?

Reply:
yes, in the getopt object